third_party/terraform/utils/tpu_operation.go

```diff
@@ -0,0 +1,29 @@
+package google
+
+import (
+	"fmt"
+
+	tpu "google.golang.org/api/tpu/v1"
+)
+
+type TpuOperationWaiter struct {
```

Comment:
@rambleraptor: When this is merged this could be another good candidate for early autogen.